docker/Dockerfile

```diff
@@ -21,7 +21,8 @@ RUN apk add --no-cache --virtual .build-deps gcc git libc-dev make libffi-dev li
     GOBIN=/usr/local/bin/ go install -modfile go.mod github.com/projectdiscovery/nuclei/v3/cmd/nuclei && \
     cd / && \
     git clone https://github.com/rfc-st/humble.git --branch master /humble && \
-    pip install --no-cache-dir -r requirements.txt -r /humble/requirements.txt $ADDITIONAL_REQUIREMENTS && \
+    git clone https://github.com/inc0d3/moodlescan.git /moodle_scanner && \
```

Comment:
this repository is licensed under GPL, and as such should be part of https://github.com/cert-polska/artemis-modules-extra package, as it's not compatible with BSD, the license of https://github.com/cert-polska/artemis